media-libs/libilbc/libilbc-1.1.1.ebuild

```diff
@@ -16,7 +16,7 @@ fi
 inherit eutils multilib ${SCM}
 
 DESCRIPTION="Packaged version of iLBC codec from the WebRTC project"
-HOMEPAGE="https://github.com/dekkers/libilbc"
+HOMEPAGE="https://github.com/lu-zero/libilbc"
```

Comment:
I'll add the current TimothyGu repository as a second HOMEPAGE to the lu-zero repository. It seems like that for v1 of this library, lu-zero might have modified the upstream code to make it easier to package. Therefore lu-zero's repository is really the HOMEPAGE for the modifications/changes, but TImothyGu's repository is the HOMEPAGE for the upstream project.